infra/build/developer-tools-krm/Dockerfile

```diff
@@ -20,6 +20,13 @@ RUN apk update && apk add --no-cache openrc docker-cli docker screen nodejs-curr
 # Additional go tooling
 RUN GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.1
 
+# Required to download and install asmcli
+ARG ASMCLI_VERSION
+ENV ASMCLI_VERSION ${ASMCLI_VERSION}
+
+ADD ./build/install_asmcli.sh /build/
+RUN /build/install_asmcli.sh ${ASMCLI_VERSION}
+
```

Comment:
We can remove /build since we wont need it after install

Reply:
is this what you mean?
```
ADD ./build/install_asmcli.sh .
RUN install_asmcli.sh ${ASMCLI_VERSION}
```
Having the install script at root doesn't seem right. Just to be clear this isn't install ASM yet. This is only setting up the utility to install ASM later.

Reply:
No, I meant 

```
ADD ./build/install_asmcli.sh /build/
RUN /build/install_asmcli.sh ${ASMCLI_VERSION}
RUN rm -rf /build
```
This way `/build/install_asmcli.sh` is not persisted as we dont need it after the CLI is installed.

> Having the install script at root doesn't seem right. 

Yeah we can continue to keep the script in /build which we can then clean up.

> Just to be clear this isn't installing ASM yet. This is only setting up the utility to install ASM later.

Yup, understood that this is just installing the cli and not installing ASM.

Reply:
Got it. I'll remove that.